lib/puppet/parser/functions/ssh_global_known_hosts.rb

```diff
@@ -5,6 +5,8 @@ module Puppet::Parser::Functions
       require 'find'
```

Comment:
Because this function is getting an error when it runs.  Trevor is asking that this be gutted and just spit out a a deprication worning so it does not fail but does not do anything.